src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -233,6 +244,26 @@ class HttpGoogleServicesDAO(
       }) { case t: HttpResponseException if t.getStatusCode == 409 => bucketName }
   }
 
+  def createStorageLogsBucket(billingProject: RawlsBillingProjectName): Future[String] = {
+    val bucketName = getStorageLogsBucketName(billingProject)
+    logger debug s"storage log bucket: $bucketName"
+
+    retryWithRecoverWhen500orGoogleError(() => {
+      val bucket = new Bucket().setName(bucketName)
+      val deleteAfterOneYear = new Lifecycle.Rule()
+        .setAction(new Action().setType("Delete"))
+        .setCondition(new Condition().setAge(365))
```

Comment:
Maybe a year is too much, maybe it's too little. Seems there should be some limit on it, though. Other suggestions welcome.